templates/actors/character-sheet.hbs

```diff
@@ -223,18 +220,15 @@
                         <h4 class="rollable" data-action="rollDeathSave">{{ localize "DND5E.DeathSave" }}</h4>
                         <div class="counter-value">
                             <i class="fas fa-check"></i>
-                            {{numberInput system.attributes.death.success name="system.attributes.death.success"
-                                placeholder="0"}}
+                            <input type="text" value="{{system.attributes.death.success}}" name="system.attributes.death.success" class="delta-input" placeholder="0">
```

Comment:
I'm not sure delta inputs are really necessary for death saves, because generally that is handled by the core roller. They weren't enabled here before the recent changes anyway.

Reply:
I will do this after my uni homework tonight.

Reply:
I mean, that's true but I guess I don't see why we wouldn't do this unless it breaks something else. I tried to more or less make all inputs on the sheet that use only numbers compatible.

Reply:
Its fine by me :)